nixos/modules/services/networking/nordvpn.nix

```diff
@@ -0,0 +1,79 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.nordvpn;
+in {
+  options = {
+    services.nordvpn = {
+      enable = mkEnableOption "nordvpn";
+
+      package = mkOption {
```

Comment:
Will this option ever actually be used? :thinking:

Reply:
my understanding is that is good to provide a cfg.package to allow swapping packages

Reply:
If a package is configurable (different feature flags, versions, etc...) it is a great idea to add this option. If there are no options at all, maybe overlays are what you really want. Your call.